src/plugins/console/server/lib/spec_definitions/json/overrides/indices.put_template.json

```diff
@@ -5,7 +5,9 @@
       "mappings": { "__scope_link": "put_mapping" },
       "settings": { "__scope_link": "put_settings" },
       "version": 0,
-      "order": 0,
+      "template": {},
```

Comment:
This file is for the [legacy index template API](https://www.elastic.co/guide/en/elasticsearch/reference/master/indices-templates-v1.html).

We need to add an overrides file for the [composable index templates API](https://www.elastic.co/guide/en/elasticsearch/reference/master/index-templates.html). Those docs contain some bugs and will be fixed by the ES Docs team. In the meantime, we just need to undo this change, and create a new `indices.put_index_template.json` override file with this content:

```json
{
  "indices.put_index_template": {
    "data_autocomplete_rules": {
      "index_patterns": [],
      "version": 0,
      "template": {
        "mappings": { "__scope_link": "put_mapping" },
        "settings": { "__scope_link": "put_settings" },
        "aliases": {
          "__template": {
            "NAME": {}
          }
        }
      },
      "composed_of": [],
      "priority": 0,
      "_meta": {}
    },
    "patterns": [
      "_index_template/{template}"
    ]
  }
}

```

Reply:
Done.